components/compliance-service/reporting/relaxting/reports.go

```diff
@@ -1163,14 +1192,28 @@ func (backend ES2Backend) getFiltersQuery(filters map[string][]string, latestOnl
 
 	if latestOnly {
 		// only if there is no job_id filter set, do we want the daily latest
+		s, err := filterQuerychange(filters)
+		if err != nil {
+			panic(err)
+		}
 		if len(filters["end_time"]) > 0 {
-			// If we have an end_time filter, we use the daily_latest filter dedicated to the UTC timeseries indices
-			termQuery := elastic.NewTermsQuery("daily_latest", true)
-			boolQuery = boolQuery.Must(termQuery)
+			if len(s) == 1 && s[0] == "daily_latest" {
```

Comment:
This complete code can be moduled out to a different function or at least a loop.

```
for search_field := range s {
      termQuery := elastic.NewTermsQuery(search_field, true)
      boolQuery = boolQuery.Must(term query)
}
```